update-testsuite.sh

```diff
@@ -80,6 +85,8 @@ merge_with_spec() {
 
     set_upstream ${repo}
 
+    [ "${repo}" == "wasm-3.0" ] && return
```

Comment:
Can you add a comment here describing why we early return in this case?

Reply:
In commenting this I realized it was actually easier to just go ahead and handle this, so it no longer early-returns